components/card/README.md

```diff
@@ -292,17 +292,22 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
 | Property | Type | Description |
 |--|--|--|
 | `icon` | String, required | Preset icon key (ex. "tier1:gear") |
-| `text` | String, required | Accessible text for the link (not visible, gets announced when user focuses) |
```

Comment:
This is still required here as it's used within the inner `d2l-count-badge-icon`

Reply:
Yeah, looks like I've overcorrected 😅 I'll change that back!